types/object-hash/index.d.ts

```diff
@@ -105,7 +105,7 @@ declare namespace objectHash {
 /**
  * @see https://github.com/puleos/object-hash#hashvalue-options
  */
-declare function objectHash(object: {} | null, options?: objectHash.NormalOption): string;
```

Comment:
can it be `undefined`?
https://github.com/puleos/object-hash/blob/master/index.js#L81

Reply:
No, It can't but what should I use as type if I don't want to use `undefined`? Because `{}` is not a type. Am I right?

Reply:
probably `NotUnefined` (with better name): 
[playground](https://www.typescriptlang.org/play?ts=4.8.4#code/C4TwDgpgBAcg9sAqgOwCYQGYEtkVVAXigG8BfKAHymQFcAbOgbgFgAodAYzoEMAnaDDWQdgWOMihwARgCsIIgBLcAzgAsAFAA8AXLAQp02XKgCUugG5wsqFqzbS5ilRoAMJxlE9fvUAPS-JAGt7WXlgJTV1ACIo9x8ffyCQx3DndWBeGgg4+L8AuGDWBzCIjTIc3LykotCnSNoGCoT8wuK6jQBtFwBdJubqttTIoUMcPArEiF5eOF4gA)

Reply:
Thanks, updated